x-pack/plugins/security_solution/public/common/components/landing_page/index.tsx

```diff
@@ -8,11 +8,14 @@
 import React, { memo, useMemo } from 'react';
 import { useSourcererDataView } from '../../containers/sourcerer';
 import { getOnboardingComponent } from './onboarding';
+import type { StepId } from './onboarding/types';
 
-export const LandingPageComponent = memo(() => {
-  const { indicesExist } = useSourcererDataView();
-  const OnBoarding = useMemo(() => getOnboardingComponent(), []);
-  return <OnBoarding indicesExist={indicesExist} />;
-});
+export const LandingPageComponent = memo(
+  ({ defaultExpandedStep }: { defaultExpandedStep?: StepId }) => {
+    const { indicesExist } = useSourcererDataView();
+    const OnBoarding = useMemo(() => getOnboardingComponent(), []);
+    return <OnBoarding indicesExist={indicesExist} defaultExpandedStep={defaultExpandedStep} />;
```

Comment:
Do we need this `getOnboardingComponent` function now that we have the component? Couldn't we just `import { Onboarding } from './lazy';` and render it directly here? 🤔

Reply:
You are right @semd , we don't need this anymore. I've removed it in https://github.com/elastic/kibana/pull/175719/files#diff-10832002bc5dfd1354f367e187c839c6e69e7155931e0bf2ab2db4e1fb6afba4